examples/integration_with_error.py

```diff
@@ -1,151 +1,124 @@
 """
-Check Integration
-=================
+Check Integration_w_error Function
+==================================
 
-Makes sure that automatically chosen integral bounds perform similar to or
+Generates a fake dataset of an inversion recovery w/ normally distributed
+random noise.
+Then checks that automatically chosen integral bounds perform similar to or
```

Comment:
honestly, this seems to be doing a similar job to check_integration.py but figured we could discuss about deleting later and so I just updated this one